index.html

```diff
@@ -0,0 +1,158 @@
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+        type="text/css" />
+    <title>Microverse Capstone One</title>
+</head>
+
+<body>
+    <header>
+        <nav class="header-menu">
+            <a href="#" class="mx-3 text-white"><i class="fab fa-facebook"></i></a>
+            <a href="#" class="mx-3 text-white"><i class="fab fa-twitter"></i></a>
+            <a href="#" class="mx-3 text-white">English</a>
+            <a href="#" class="mx-3 text-white">MyPage</a>
+            <a href="#" class="mx-3 text-white">Logout</a>
+        </nav>
+        <nav class="desktop-menu">
+                <a class="os-logo" href="index.html">
+                    <i class="fab fa-osi"></i>
+                    <span>JavaScript Society <br></span>
+                    <span>Submit 2023</span>
+                </a>
+                <ul class="nav-list">
+                <li><a class="about" href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="OS-campaign" href="#">OS Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="mobile-top-page">
+        <div class="mobile-top-pag-bg">
+            <div class="menu1">
+                <span class="menu"><a href="about.html"></a></span>
+                <span class="menu"><a href="about.html"></a></span>
+                <span class="menu"><a href="about.html"></a></span>
+            </div>
+        </div>
+        <div class="mobile-menu">
+            <div class="sidebar-cancel">
+                <i class="far fa-times-circle fa-3x closeBtn"></i>
+            </div>
+            <div class="menuPopUp">
+                <a href="index.html" class="anchor-link">Home</a>
+                <a href="about.html" class="anchor-link">About</a>
+                <a href="index.html" class="anchor-link">Program</a>
+                <a href="index.html" class="anchor-link">Join</a>
+                <a href="index.html" class="anchor-link">Sponsor</a>
+                <a href="index.html" class="anchor-link">News</a>
+            </div>
+        </div>
+        <main>
+            <section id="program">
+                <div class="mobile-first-page">
+                    <h1>"Welcome To"</h1>
+                    <h2>JavaScript Society<br>Global Summit<br>2023</h2>
+                    <p>An Javascript coders community that believes in the value of openness, sharing, creativity and tech
+                        empowerment. Creating a positive change with people all over the world.
+                        We are hosting Africa biggest open source conference this November, in Nairobi, Kenya.</p>
+                    <h3>2023.07.5(WED) ~ 7(FRI)</h3>
+                    <h4>@ CC-hub Yaba, Nairobi, Kenya</h4>
```

Comment:
- [ ] Please align the content of your first section according to the provided design as shown in the image below and also make sure that the text color is in contrast with the background color so that the text is readable for example the text color of the date should be lighter or brighter to make it stand out on a dark background.

Cindy's design
----
<img width="675" alt="Screenshot 2023-07-07 at 8 45 19 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/938a4f18-4284-4df5-93df-ff060e8f3aa8">


Your design
-----
<img width="1508" alt="Screenshot 2023-07-07 at 6 26 27 PM" src="https://github.com/peter-mwovi/Capstone-project/assets/44798044/01161d64-4b8f-4585-a874-2ff0071bf83d">